sr_robot_launch/launch/sr_ur5arm_box.launch

```diff
@@ -31,10 +31,10 @@
   <arg name="joint_state_pub_frequency" default="125"/>
 
   <!-- HAND CONFIG-->
-  <arg name="hand_id" default="H1"/>
+  <arg name="hand_prefix" default="H1"/>
 
   <!-- ARM CONFIG-->
-  <arg name="arm_id" default="ra"/>
+  <arg name="arm_prefix" default="ra"/>
```

Comment:
why not have arm_prefix take the first letter of side as you do in other launch files?